### ISSUE TITLE
Print views are incomplete

### ISSUE THREAD

**Issue by ChristinaSi** [dboehmer/coocook#94]
![screenshot_printing_incomplete](https://user-images.githubusercontent.com/17025347/50398605-36014180-0779-11e9-85c2-f781700ccf8e.png)
![screenshot_printing_strange](https://user-images.githubusercontent.com/17025347/50398606-36014180-0779-11e9-8b44-e17cef1127d9.png)
As you can see in the attached files some print views are incomplete, some have a strange footer and some are totally okay. When pressing Strg + P the resulting pdf files are all complete, it is just the view in our app that is not completely working.
The views are from project Jugule-Silvester 2018/19.

**Comment by dboehmer** [comment]
So this is only a bug report but your project works? Do you need immediate support?

Merry Christmas everyone!

**Comment by ChristinaSi** [comment]
No, immediate support is not necessary, enjoy your christmas holidays.

**Comment by MarkusLeupold** [comment]
I think, this issue has been fixed since Oct 2018 by af33884

**Comment by MarkusLeupold** [comment]
Yay, time travel! :D The issue was written after the commit I linked in the previous comment. But still, I am quite sure, that this problem doesn't occur anymore.

**Comment by dboehmer** [comment]
> Yay, time travel! :D The issue was written after the commit I linked in the previous comment. But still, I am quite sure, that this problem doesn't occur anymore.

I’m not quite sure if the mentioned commit was already merged and deployed on Dec 24, 2018. If the commit actually solves the issue, this would explain the time glitch;-) Still need to check that.
